app/models/competition.rb

```diff
@@ -382,6 +384,25 @@ def advancement_condition_must_be_present_for_all_non_final_rounds
     end
   end
 
+  validate :auto_accept_validations
+  private def auto_accept_validations
+    errors.add(:auto_accept_registrations, I18n.t('competitions.errors.must_use_wca_registration')) if
+      auto_accept_registrations && !use_wca_registration
```

Comment:
I think you can use `auto_accept_registrations?` here, to make it more explcitly clear that you're checking for (non-zero) presence

Reply:
Changed